assets/js/WfModule.js

```diff
@@ -336,6 +336,13 @@ class WfModule extends React.Component {
                 />
               </div>
 
+    var helpIcon = undefined;
```

Comment:
`var helpicon` would also set to undefined. If you intend no value, generally `var helpicon=null` is the way to go.

Reply:
Noted, will change where I find this elsewhere as well.